pkg/auth/context_test.go

```diff
@@ -0,0 +1,152 @@
+package auth
+
+import (
+	"testing"
+
+	"github.com/golang-jwt/jwt/v4"
+	"github.com/onsi/gomega"
+)
+
+func TestContext_GetAccountIdFromClaims(t *testing.T) {
+	tests := []struct {
+		name   string
+		claims jwt.MapClaims
+		want   string
+	}{
+		{
+			name:   "Should return empty when tenantUserIdClaim is empty",
+			claims: jwt.MapClaims{},
+			want:   "",
+		},
+		{
+			name: "Should return when tenantUserIdClaim is not empty",
+			claims: jwt.MapClaims{
+				tenantUserIdClaim: "Test_user_id",
+			},
+			want: "Test_user_id",
+		},
+	}
+
+	for _, tt := range tests {
+		gomega.RegisterTestingT(t)
+		t.Run(tt.name, func(t *testing.T) {
+			accountId := GetAccountIdFromClaims(tt.claims)
+			gomega.Expect(tt.want).To(gomega.Equal(accountId))
```

Comment:
If we change the import from 
```
"github.com/onsi/gomega"
```
to
```
. "github.com/onsi/gomega"
```
we would be able to write this code as:
```go
Expect(tt.want).To(Equal(accountId))
```

making it more readable. WDYT?